src/common.h

```diff
@@ -187,16 +187,27 @@ inline static unsigned long long GetNanoPerformanceTime()
 #endif
 
 #define GETCONTROLLERIMPL() ControllerClientImplPtr controller = boost::dynamic_pointer_cast<ControllerClientImpl>(GetController());
-#define CHECKCURLCODE(code, msg) if (code != CURLE_OK) { \
-        throw MujinException(boost::str(boost::format("[%s:%d] curl function %s with error '%s': %s")%(__PRETTY_FUNCTION__)%(__LINE__)%(msg)%curl_easy_strerror(code)%_errormessage), MEC_HTTPClient); \
+#define CHECKCURLCODE(code, msg) { \
```

Comment:
Please put the comment outside of the define.

Please use `do { ... } while (false)` to scope instead of just `{ ... }`.

Reply:
@ziyan I edited the scope to the do { ... } while (false) form. Can you please check when you are free? Thanks.